### DISCUSSION TITLE
Module paths aliases NOT integrating automatically with vscode

### DISCUSSION THREAD

**Discussion by tomasgil123** [vercel/next.js#15798] [Help]
Im trying to use absolute paths with Nextjs, but vscode complains it can not resolve the module. Nevertheless, the site is working ok.

My jsconfig.json file:
![image](https://user-images.githubusercontent.com/14185059/89124856-83df3f00-d4b0-11ea-8e10-f3be35b1acc7.png)

How Im trying to import the component:
![image](https://user-images.githubusercontent.com/14185059/89124879-ae30fc80-d4b0-11ea-9685-c747dc9faa0d.png)

My folder structure:
![image](https://user-images.githubusercontent.com/14185059/89124894-cd2f8e80-d4b0-11ea-8f8b-8d8e31dca81d.png)

Vscode error message:
![image](https://user-images.githubusercontent.com/14185059/89124906-e2a4b880-d4b0-11ea-93d3-65c4d88fc8bc.png)

**Comment by Oikio** [comment]
It's not vscode what is complaining, as far as I can tell - it's eslint, just add paths to eslint.json, something like:
```
{
...
  "parserOptions": {
    "project": ["./packages/*/jsconfig.json", "./jsconfig.*"]
  },
}
